src/python/bot/tasks/minimize_task.py

```diff
@@ -1244,6 +1248,40 @@ def handle_unreproducible():
   expected_state = initial_crash_result.get_symbolized_data()
   logs.log('Initial crash state: %s\n' % expected_state.crash_state)
 
+  # Minimize *_OPTIONS env variable first.
+  env = {}
+  for tool in environment.SUPPORTED_MEMORY_TOOLS_FOR_OPTIONS:
+    options_env_var = tool + '_OPTIONS'
+    options = environment.get_memory_tool_options(options_env_var)
+    if not options:
+      continue
+
+    minimized_options = copy.deepcopy(options)
+    for option_name, option_var in six.iteritems(options):
+      minimized_options.pop(option_name)
+      environment.set_memory_tool_options(options_env_var, minimized_options)
+
+      _, crash_result = _run_libfuzzer_tool(
```

Comment:
should we be calling libfuzzer with minimize here? can we just run the testcase without minimizing (i.e. by calling _run_libfuzzer_testcase)? seems weird to do it here and then again from lines 1287 onwards.